pkg/cli/environment.go

```diff
@@ -131,6 +149,8 @@ func envCSV(name string) (ls []string) {
 	return
 }
 
+// EnvVars reads the hypper and helm enviromental variables and populates
+// values in the EnvSettings caller.
 func (s *EnvSettings) EnvVars() map[string]string {
 	envvars := map[string]string{
 		"HELM_BIN":               os.Args[0],
```

Comment:
What is this used for?

Reply:
So far, it's only used for resetting the environment after tests, in https://github.com/rancher-sandbox/hypper/blob/viccuad%2Fadd-annotations/pkg/cli/environment_test.go#L120.
It seems that it would be used more in plugins and cmd/helm/env.go.

Reply:
:+1: